lib/bridge.js

```diff
@@ -672,6 +679,48 @@ Bridge.prototype._updateIntents = function(event) {
     });
 };
 
+/**
+ * Returns a PrometheusMetrics instance. The instance will be registered with
+ * the HTTP server so it can serve the "/metrics" page in the usual way.
+ */
+Bridge.prototype.getPrometheusMetrics = function() {
```

Comment:
It's not clear that this will implicitly *create* a `PrometheusMetrics`. Please mention this in the docs.